pace-util/pace/util/comm.py

```diff
@@ -63,3 +63,7 @@ def Irecv(self, recvbuf, source, tag: int = 0, **kwargs) -> Request:
     @abc.abstractmethod
     def Split(self, color, key) -> "Comm":
         ...
+
+    @abc.abstractmethod
+    def allreduce(self, sendobj, op=None) -> Any:
```

Comment:
`None` is not a valid argument for `op`, unfortunately. The default value is `MPI.SUM`, which is only available if we can import from mpi4py, but I think we can get away with the built-in `sum` object as default.
```suggestion
    def allreduce(self, sendobj, op=sum) -> Any:
```

I'd also be fine with making this a required argument on our end.

Reply:
none is a valid argument for op